src/webgpu/api/validation/encoding/queries/begin_end.spec.ts

```diff
@@ -56,7 +77,23 @@ Tests the invalid nesting of begin/end occlusion queries:
         { calls: [0, 1, 'end', 'end'] },
```

Comment:
(see below) Can add a private param here so you don't need a "magic" check to determine whether the case is valid or not.

```
        { _ok: true, calls: [0, 'end', 1, 'end'] },
        { _ok: false, calls: [0, 0, 'end', 'end'] },
        { _ok: false, calls: [0, 1, 'end', 'end'] },
```

Reply:
Thanks, update to use private parameter and use for loop instead.